lib/ui.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+const execa = require('execa');
+const inquirer = require('inquirer');
+const readPkgUp = require('read-pkg-up');
+const chalk = require('chalk');
+const version = require('./version');
+
+module.exports = options => {
+	const pkg = readPkgUp.sync().pkg;
+	const oldVersion = pkg.version;
+
+	console.log('\n');
+	console.log(`Releasing a new version of ${chalk.bold(pkg.name)} ${chalk.gray.dim(`(current version: ${oldVersion})`)}`);
+	console.log('\n');
+
+	const prompts = [
+		{
+			type: 'list',
+			name: 'version',
+			message: 'Select semver increment or specify new version',
+			choices: version.SEMVER_INCREMENTS
+				.map(inc => ({
+					name: `${inc} ${chalk.gray.dim(`(${version.getNewVersion(oldVersion, inc)})`)}`,
+					value: inc
+				}))
+				.concat([
+					new inquirer.Separator(),
+					{
+						name: 'Other (specify)',
+						value: null
+					}
+				]),
+			filter: input => version.isValidVersionInput(input) ? version.getNewVersion(oldVersion, input) : input
+		},
+		{
+			type: 'input',
+			name: 'version',
+			message: 'Version',
+			when: answers => !answers.version,
+			filter: input => version.isValidVersionInput(input) ? version.getNewVersion(pkg.version, input) : input,
+			validate: input => {
+				if (!version.isValidVersionInput(input)) {
+					return 'Please specify a valid semver, e.g. 1.2.3. See http://semver.org';
+				} else if (!version.isVersionGreater(oldVersion, input)) {
+					return `Version must be greater than ${oldVersion}`;
+				}
+				return true;
+			}
+		},
+		{
+			type: 'list',
+			name: 'tag',
+			message: 'How should this pre-release version be tagged in npm?',
+			when: answers => !pkg.private && version.isPrereleaseVersion(answers.version) && !options.tag,
+			choices: () => {
+				return execa.stdout('npm', ['dist-tag', 'ls'])
```

Comment:
Return `execa` immediately behind the arrow function

`choices: () => execa.stdout()`